scripts/docs-generation/wdioDocs.js

```diff
@@ -1,18 +1,9 @@
 const fs = require('fs')
 const path = require('path')
-const markdox = require('markdox')
-const { promisify } = require('util')
-
-const formatter = require('../utils/formatter')
-const compiler = require('../utils/compiler')
+const jsdoc2md = require('jsdoc-to-markdown')
 const TEMPLATE_PATH = path.join(__dirname, '..', 'templates', 'api.tpl.ejs')
```

Comment:
What happened to this template file? How do we ensure that the docs are still in the right format?

Reply:
Made changes..!!
Please have a look

Reply:
Added 
```
const JSDOC_OPTIONS = {
    formatter: formatter,
    compiler: compiler,
    template: TEMPLATE_PATH
}
```
and included the options 
```const out = await processDocs({data:[filepath], options:JSDOC_OPTIONS}) ```

These must ensure the generated ```.md``` files are in the right format..!!

Let me know if any changes are to be done!

Reply:
@abhishek-bodapati can you build the page and send a screenshot?

Reply:
I had run ```node generateDocs.js ``` to generate ```.md``` files from ```.ts``` files. Build was successful. Here are the screenshots 
![Screenshot 2021-05-25 at 7 44 18 PM](https://user-images.githubusercontent.com/47183749/119513222-a6b94780-bd91-11eb-96d9-9dc9c40edf74.png)
![Screenshot 2021-05-25 at 7 44 28 PM](https://user-images.githubusercontent.com/47183749/119513243-aae56500-bd91-11eb-8dd7-a2cd38e302e7.png)